app-nodejs-html5/README.md

```diff
@@ -0,0 +1,80 @@
+app-nodejs-html5: HTML5 Service Invocation Application
+===================================================
+
+Level: Beginner  
+Technologies: HTML5, JavaScript  
+Summary: HTML5 Service Invocation Application run under Express  
+Target Product: Keycloak  
+Source: <https://github.com/keycloak/keycloak-quickstarts>
+
+What is it?
+-----------
+
+The `app-nodejs-html5` quickstart demonstrates how to write an application with HTML5 and JavaScript that authenticates
+using <span>Keycloak</span>. Once authenticated the application shows how to invoke a service secured with <span>Keycloak</span>.
+
+System Requirements
+-------------------
+
+You need to have Node.js version 12.x or later installed.
+
+The quickstart requires that you have the [service-nodejs](../service-nodejs/README.md) running. It assumes the
+services are located at `http://localhost:3000/service`. If the services are running elsewhere you need to edit
+`app.js` and replace the value of `serviceUrl`.
+
+Configuration in <span>Keycloak</span>
+-----------------------
+
+Prior to running the quickstart you need to create a client in <span>Keycloak</span> and download the installation file.
+
+The following steps show how to create the client required for this quickstart:
+
+* Open the <span>Keycloak</span> admin console
```

Comment:
Why put 'Keycloak' in a span element? Seems like that would render the same as any other text in the readme, so perhaps it can be omitted?

```suggestion
using Keycloak. Once authenticated the application shows how to invoke a service secured with Keycloak.

System Requirements
-------------------

You need to have Node.js version 12.x or later installed.

The quickstart requires that you have the [service-nodejs](../service-nodejs/README.md) running. It assumes the
services are located at `http://localhost:3000/service`. If the services are running elsewhere you need to edit
`app.js` and replace the value of `serviceUrl`.

Configuration in Keycloak
-----------------------

Prior to running the quickstart you need to create a client in Keycloak and download the installation file.

The following steps show how to create the client required for this quickstart:

* Open the Keycloak admin console
```

Reply:
@jonkoops much of that text was just copied from the Java version of the example but happy to remove the span as I'm not sure why it was needed.